symbionic/gui/kivy_gui.py

```diff
@@ -148,6 +156,29 @@ def save(self, path, filename):
         self.dismiss_popup()
 
 
+class clsCheckBox:
+    def __init__(self, Logic, cbox_id, number_of_cbs, CB_group):
+        self.Logic = Logic
+        self.cbox_id = cbox_id
+        self.number_of_cbs = number_of_cbs
+        self.CB_group = CB_group
+        self.cbs = []
+        self.ylim = (-150,150)
+
+    def init_cbs(self):
+        for g in range(self.number_of_cbs):
+            self.add_cb(self.cbox_id)
+
+    def add_cb(self, box_id):
+        cbs = CheckBox(group=self.CB_group)
+        cbs.bind(active=self.on_checkbox_active)
+        self.cbs.append(cbs)
+        box_id.add_widget(cbs)
+
+    def on_checkbox_active(self, checkboxId, isActive):
+        if isActive:
+           self.Logic.selected_gesture = str(self.cbs.index(checkboxId) + 1)
```

Comment:
Is it smart that we have a dependency on the Logic class (which I renamed to Root..sorry)? I have to think about this.

Reply:
let's discuss this Friday. Logic is changed to root now as well